modin/core/storage_formats/base/query_compiler.py

```diff
@@ -1320,6 +1320,31 @@ def to_numeric(self, *args, **kwargs):  # noqa: PR02
         """
         return SeriesDefault.register(pandas.to_numeric)(self, *args, **kwargs)
 
+    @doc_utils.add_one_column_warning
+    @doc_utils.add_refer_to("to_timedelta")
+    def to_timedelta(self, unit="ns", errors="raise", *args, **kwargs):  # noqa: PR02
+        """
+        Convert argument to timedelta.
+
+        Parameters
+        ----------
+        unit : str, default: "ns"
+            Denotes the unit of the arg for numeric arg. Defaults to "ns".
+        errors : {"ignore", "raise", "coerce"}, default: "raise"
+        *args : iterable
+            Serves the compatibility purpose. Does not affect the result.
+        **kwargs : dict
+            Serves the compatibility purpose. Does not affect the result.
```

Comment:
```suggestion
    def to_timedelta(self, unit="ns", errors="raise"):
        """
        Convert argument to timedelta.

        Parameters
        ----------
        unit : str, default: "ns"
            Denotes the unit of the arg for numeric arg. Defaults to "ns".
        errors : {"ignore", "raise", "coerce"}, default: "raise"
```

I don't think pandas supports `*args` or `**kwargs`